### PR TITLE
Equivalence relation

### PR DESCRIPTION
**Pull request by stefan-aws**
By submitting this pull request, I confirm that my contribution is made under the terms of the MIT license.

### CODE REVIEW

File: examples/Relations.dfy [hunk 1]
```diff
@@ -18,4 +18,15 @@ module RelationsExamples {
   lemma BuiltInIntLEIsTotalOrdering()
     ensures TotalOrdering((x: int, y: int) => x <= y)
   {}
+
+  function Modulo(n: nat): (R: (int, int) -> bool)
+    requires n > 1
```
Comment (davidcok) [inline examples/Relations.dfy:23]: Why not allow 1?

**Review by davidcok — CHANGES_REQUESTED** [review]
Be sure to add some examples that serve as tests.

File: examples/Relations.dfy (hunk 1 above)
Comment (stefan-aws) [reply examples/Relations.dfy]: Fixed

**Review by jtristan — APPROVED** [review]

**Review by davidcok — APPROVED** [review]
Since John has already approved, I'll add an approval.
But please add this change in src/dafny also -- and make any future changes there.

**Pull request merged**